From ab8cb80f941a8f8242feb7877e8f7fbf600873be Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Fri, 27 Nov 2015 11:10:53 +0800
Subject: [PATCH 0699/5242] MLK-11915-03 ASoC: imx-sii902x: init sii902x_np to
 avoid wild pointer

commit  e2b2fbf5d86a79cdf6b0c7fade9bf2725ff04420 from
https://source.codeaurora.org/external/imx/linux-imx.git

init sii902x_np to avoid wild pointer. Reported by Coverity.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
(cherry picked from commit 780e27683c38f785ea7f7e07e83b00ffd3c22d78)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/imx-sii902x.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/fsl/imx-sii902x.c b/sound/soc/fsl/imx-sii902x.c
index 72c407d..7cd19fa 100644
--- a/sound/soc/fsl/imx-sii902x.c
+++ b/sound/soc/fsl/imx-sii902x.c
@@ -172,7 +172,7 @@ static void imx_sii902x_shutdown(struct snd_pcm_substream *substream)
 
 static int imx_sii902x_probe(struct platform_device *pdev)
 {
-	struct device_node *cpu_np, *sii902x_np;
+	struct device_node *cpu_np, *sii902x_np = NULL;
 	struct platform_device *cpu_pdev;
 	struct imx_sii902x_data *data;
 	int ret;
-- 
1.7.9.5

